blocks/hooks/utils.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { isString } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Children, cloneElement } from '@wordpress/element';
+
+export function extendElement( element, newElement ) {
```

Comment:
This is nice and it seems very useful. We could add unit tests. I'm wondering if snapshot approach would fit here. I can play later with it, as we miss snapshot serializer for `enzyme` at the moment.

Reply:
If we wanted to support prepending to an existing element, could we make this more generic to `extendElement( ...elements )` like an `Object.assign` ? Maybe closer to a `concatChildren` behavior in that case.

Reply:
And to the above point, is this really a hook utility? Or something more appropriate for `wp.element`?

Reply:
In fact... isn't this achieving the same behavior as `concatChildren` ?